README.md

```diff
@@ -1,58 +1,50 @@
-# commavq
-commaVQ is a dataset of 100,000 heavily compressed driving videos for Machine Learning research. A heavily compressed driving video like this is useful to experiment with GPT-like video prediction models. This repo includes an encoder/decoder and an example of a video prediction model.
 
-## 2x$1000 Challenges!
+| Source Video    | Compressed Video | Future Prediction |
```

Comment:
github padding makes the videos tiny. bet we can do better building the table in HTML